app/app.go

```diff
@@ -192,9 +193,33 @@ func NewOsmosisApp(
 		panic(fmt.Sprintf("error while reading wasm config: %s", err))
 	}
 
-	app.InitSpecialKeepers(skipUpgradeHeights, homePath, invCheckPeriod)
+	app.InitSpecialKeepers(
+		appCodec,
+		bApp,
+		keys,
+		tkeys,
+		memKeys,
```

Comment:
We should be able to generate these in the keepers package right? Perhaps we have a new file in there called storekeys, which has functions for these?

Reply:
yes, i think generate these keys in keepers package is better.

Reply:
i think  transfer module should be in keepers package instead of what i did.